packages/git/src/browser/git-scm-provider.spec.ts

```diff
@@ -109,12 +109,7 @@ describe('GitScmProvider', () => {
     });
 
     afterEach(async () => {
-        await new Promise<void>((resolve, reject) => rimraf(FileUri.fsPath(repository.localUri), error => {
-            if (error) {
-                reject(error);
-            }
-            resolve();
-        }));
+        rimraf(FileUri.fsPath(repository.localUri));
```

Comment:
For this function to behave as before, the `rimraf` needs to be awaited.
```suggestion
        await rimraf(FileUri.fsPath(repository.localUri));
```